packages/framework/src/Model/Order/Item/OrderItemFactory.php

```diff
@@ -63,6 +63,36 @@ public function createProduct(
         return $orderProduct;
     }
 
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Order\Order $order
+     * @param \Shopsys\FrameworkBundle\Model\Order\Item\OrderItemData $orderItemData
+     * @return \Shopsys\FrameworkBundle\Model\Order\Item\OrderItem
+     */
+    public function createProductFromOrderItemData(Order $order, OrderItemData $orderItemData): OrderItem
```

Comment:
Just detail - when method name includes "from order item data" I would expect it to be first argument

Reply:
```suggestion
    public function createProductOrderItemFromOrderItemData(Order $order, OrderItemData $orderItemData): OrderItem
```